docs/channels.md

```diff
@@ -0,0 +1,768 @@
+:channels &channel-action [%channel nest action]
+```
+
+Delete a reply from a post
+
+[id-post](#id-post) \| [id-reply](#id-reply)
+###### %post %reply %add-react
+```hoon
+=nest [%chat our %mychat]
+=reply-action [%add-react ~2023.2.2 our %':grinning:']
+=post-action [%reply ~2023.1.1 reply-action]
+=action [%post post-action]
+:channels &channel-action [%channel nest action]
+```
+
+Add a reaction to a reply
+
+[id-post](#id-post) \| [id-reply](#id-reply) \| [react](#react)
+###### %post %reply %del-react
+```hoon
+=nest [%chat our %mychat]
+=reply-action [%del-react ~2023.2.2 our]
+=post-action [%reply ~2023.1.1 reply-action]
+=action [%post post-action]
+:channels &channel-action [%channel nest action]
+```
+
+Delete a reaction from a reply
+
+[id-post](#id-post) | [id-reply](#id-reply)
+###### %view
+```hoon
+=nest [%chat our %mychat]
+=action [%view %grid]
+:channels &channel-action [%channel nest action]
+```
+
+Set the display format for a channel
+
+[view](#view-1)
+###### %sort
+```hoon
+=nest [%chat our %mychat]
+=action [%sort %alpha]
+:channels &channel-action [%channel nest action]
+```
+
+Set the sorting mechanism for a channel
+
+[sort](#sort-1)
+###### %set-order
+```hoon
+=nest [%chat our %mychat]
+=action [%set-order `~[~2023.1.1 ~2023.2.2]]
+:channels &channel-action [%channel nest action]
+```
+
+Set manual post ordering for a channel
+
+[arranged-posts](#arranged-posts)
+###### %add-writers
+```hoon
+=nest [%chat our %mychat]
+=action [%add-writers (sy ~[%role1 %role2])]
+:channels &channel-action [%channel nest action]
+```
+
+Enable writing to a channel for a certain set of roles
+
+[sect:groups](#sectgroups)
+
+###### %del-writers
+```hoon
+=nest [%chat our %mychat]
+=action [%del-writers (sy ~[%role1 %role2])]
+
+:channels &channel-action [channel nest action]
+```
+
+Disable writing to a diary for a certain set of roles
+
+[sect:groups](#sectgroups)
+
+#### %channel-migration
+Used internally to handle migrating from the previous version's state
+
+#### Create a Group
+```hoon
+=g -build-file /=groups=/sur/groups/hoon
+
+=create :*
+  name=%mygroup
+  title='My group'
+  description='This is my group'
+  image=''
+  cover=''
+  cordon=`cordon:g`[%open ban=[ships=~ ranks=~]]
+  members=`(jug ship sect:g)`(~(put ju *(jug ship sect:g)) our %admin)
+  secret=%.n
+  ==
+
+:groups &group-create create
+```
+
+This is done with `%groups` instead of `%channels`, but it's included here since it's necessary for testing.
+### Subscriptions
+#### /unreads
+Subscribe to unread & preview information. Each fact is a `[channels unreads]`
+
+[channels](#channels) \| [unreads](#unreads)
+#### /
+"Firehose" subscription path. Each fact is an `r-channels` (response channels)
+
+[r-channels](#r-channels)
+#### /[kind]/[ship]/[name]
+(This is effectively /[nest])
+
+Similar to `/`, but only include updates for a particular channels. Each fact is an `r-channels` (response channels)
+
+[nest](#nest) \| [r-channels](#r-channels)
```

Comment:
`/[nest]`, especially written next to all these other examples, suggests a path with a single element in it. If you want to point this out, I would just write something like:

```suggestion
#### /[kind]/[ship]/[name]

Similar to `/`, but only include updates for a particular channels. Note how the path fully describes a specific [`$nest`](#nest). Each fact is an [`$r-channels`](#r-channels) (channels response).
```

For other paths, I would either repeat the same short phrasing, or omit it entirely.